phpunit/class-block-library-comment-template-test.php

```diff
@@ -120,7 +120,32 @@ function test_build_comment_query_vars_from_block_no_context() {
 				'no_found_rows'             => false,
 				'update_comment_meta_cache' => false,
 				'hierarchical'              => 'threaded',
-				'number'                    => self::$per_page,
```

Comment:
Uhm, this change seems odd to me. As `comments/inherit` is `true` by default, and the `page_comments` option is also `true` (it's defined at the top of the tests), shouldn't this test have stayed the same, keeping the pagination query vars?  Why `number`and `paged` disappeared? 🤔